code/modules/surgery/organs/internal/cyberimp/augments_arms.dm

```diff
@@ -436,12 +436,17 @@
 			log_combat(source, target, "attempted to [picked_hit_type]", "muscle implant")
 			return COMPONENT_CANCEL_ATTACK_CHAIN
 
+	var/potential_damage = punch_damage
+	var/obj/item/bodypart/attacking_bodypart = source.get_bodypart(zone)
```

Comment:
```suggestion
	var/obj/item/bodypart/attacking_bodypart = hand
```
Arm implants already track the arm they're in

Reply:
guh